src/components/taskDetails/index.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { ChangeEvent, FC } from 'react';
+import NavBar from '@/components/navBar/index';
+import TaskContainer from './TaskContainer';
+import Details from './Details';
+import classNames from './task-details.module.scss';
+import { toast, ToastTypes } from '@/helperFunctions/toast';
```

Comment:
can we sort the imports here somewhat like in the format where
the library imports are first
then components
then CSS followed by assets

Reply:
done